source/game/kart/KartDynamics.cc

```diff
@@ -15,6 +15,19 @@ KartDynamics::KartDynamics() {
 /// @addr{0x8059F6B8}
 KartDynamics::~KartDynamics() = default;
 
+/// @addr{0x805B5B68}
+/// @brief Stabilizes the bike by rotating towards the y-axis unit vector.
```

Comment:
Not a bike.

Reply:
I fixed this while you were in the middle of the PR review